internal/resources/managementclusterregistration/management_cluster_registration_test.go

```diff
@@ -0,0 +1,180 @@
+//go:build managementclusterregistration
```

Comment:
Please name the path as internal/resources/**managementcluster**/management_cluster_registaration_test.go

Folder to be renamed from managementclusterregistartion => managementcluster

Reply:
Done.